definitions/bulk-action/json/AuthorisationCaseField.json

```diff
@@ -39,6 +39,6 @@
     "CaseTypeID": "DIVORCE_BulkAction",
     "CaseFieldID": "CaseAcceptedList",
     "UserRole": "caseworker-divorce-courtadmin_beta",
-    "CRUD": "CRD"
+    "CRUD": "CRUD"
```

Comment:
Do we need ever assign any thing a `D` permission?

Reply:
Yes, the caseworker is this case is allowed to remove cases from the accepted list if required, that means its needs a D permission. Actually, they can't add to the Accepted List right now, however even if the data is unchanged, if its part of the update payload, it needs to have the U permission it seems.